code/signs-markings/signs-markings.js

```diff
@@ -510,6 +510,16 @@ $(document).on("knack-view-render.view_2904", function(event, page) {
     "GIS QA | Markings",
   );
 });
+$(document).on("knack-view-render.view_3424", function(event, page) {
+  // create large button on the home page
+  bigButton(
+    "sign-fabrication-work-orders",
+    "view_3424",
+    "https://atd.knack.com/signs-markings#sign-fabrication-work-orders/",
```

Comment:
need to update this link to `https://atd.knack.com/signs-markings#work-orders-sign-fabrication/`

Reply:
Actually this one, `https://atd.knack.com/signs-markings#work-orders-contractor/`